src/components/OSCALLoader.js

```diff
@@ -233,6 +235,19 @@ export default function OSCALLoader(props) {
     }
   }, [props.isRestMode]);
 
+  // Handle anchor link change in window url
+  useEffect(() => {
+    // TODO: Handle GET, when the hash hasn't changed & initial navigation to page.
+    // This outputs a 404 error at this point in the console & doesn't take the user
+    // to the desired section of page based on hash.
```

Comment:
I don't think this is just a generic TODO, it feels like it's in-scope for the initial implementation of this feature. Can you elaborate a bit on what's going wrong? With real examples?

Reply:
This was a note to self. This is definitely something that needs to be handled in this issue.